### ISSUE TITLE
[DECOHACK] Official Friendly Actor Definitions regarding STR_PD_YELLOWK and MTF_DEMON_DEAD

### ISSUE THREAD

**Issue by UndeadRyker** [MTrop/DoomTools#85]
Some of the friendly actor definitions are a bit misleading.

STR_PD_YELLOWK replaces the locked door message for trying to open red doors.

MTF_DEMON_DEAD replaces both 115 and 118. 115 is correct, but 118 is the dead shotgunner's corpse and the friendly definition only replaces 118.

Not much else to say here. I've attached a .ZIP that shows the before/after when parsed through DECOHACK.
Thanks in advance for DECOHACK btw! It's incredibly useful.
[example.zip](https://github.com/MTrop/DoomTools/files/9817463/example.zip)

**Comment by MTrop** [comment]
You're right - the text positions for yellow and red key doors is swapped when compared to the "object" lock messages (switches). *Oh, id....*

The `MTF_DEMON_DEAD` one is definitely due to copy-paste on my part. Or maybe I just skipped it, I dunno. There's a lot these things, haha. Thing 118's macro is now `MTF_SHOTGUNGUY_DEAD` (to correspond with `MTF_SHOTGUNGUY` for the alive one).

Thanks for the feedback! Fixed in commit 6c190f597a9d0cf9a0d8ab69e64768c883951e64.
